src/Core_Command.php

```diff
@@ -642,6 +642,8 @@ private function _multisite_convert( $assoc_args ) {
 			$assoc_args['subdomains']
 		);
 
+		$site_id = $wpdb->get_var("SELECT id FROM $wpdb->site");
```

Comment:
This can use some fool-proofing (you can never be sure with WordPress...).
```suggestion
		$site_id = $wpdb->get_var( "SELECT id FROM $wpdb->site" );
		$site_id = ( null === $site_id ) ? 1 : (int) $site_id;
```